plugin.py

```diff
@@ -24,6 +26,29 @@ class LspHtmlPlugin(NpmClientHandler):
     )
 
     @classmethod
-    def minimum_node_version(cls) -> Tuple[int, int, int]:
-        # this should be aligned with VSCode's Nodejs version
-        return (14, 0, 0)
+    def required_node_version(cls) -> str:
+        return '>=14'
+
+    def on_ready(self, api: ApiWrapperInterface) -> None:
+        session = self.weaksession()
+        if not session:
+            return
+        self.resolve_custom_data_paths(session)
+
+    def resolve_custom_data_paths(self, session: Session) -> None:
+        custom_data_paths = session.config.settings.get('html.customData')  # type: List[str]
+        resolved_custom_data_paths = []  # type: List[str]
+        for folder in session.get_workspace_folders():
+            resolved_custom_data_paths.extend([path.abspath(path.join(folder.path, p)) for p in custom_data_paths])
```

Comment:
do you think that it would be good to verify if the `path.abspath(path.join(folder.path, p))` exist and only then add it in the `resolved_custom_data_paths`?

Reply:
fixed with [97ef675](https://github.com/sublimelsp/LSP-html/pull/38/commits/97ef675c191158f3a4b0c5c4e703951ed845826c)

Reply:
added a warning at the point of requesting the contents if the file doesn't exist